e2e/tests/features/journeys/registration/registrationStage2Test.e2e.ts

```diff
@@ -0,0 +1,204 @@
+import { test } from "../../../../hooks/CustomFixtures";
+import {
+  assertComponentsHidden,
+  assertErrorUtil,
+} from "../../../../utils/visibilityUtils";
+
+test.describe("Successful user journey (UP TO OTP PAGE)", () => {
```

Comment:
Would add the .skip to this test until it's fully ready with the OTP stuff/the workaround